pkgs/by-name/sh/shadps4/package.nix

```diff
@@ -127,15 +123,17 @@ stdenv.mkDerivation (finalAttrs: {
       $out/bin/shadps4
   '';
 
-  passthru.updateScript = unstableGitUpdater {
-    tagFormat = "v.*";
-    tagPrefix = "v.";
+  passthru = {
+    tests.openorbis-example = nixosTests.shadps4;
+    updateScript = unstableGitUpdater {
+      tagFormat = "v.*";
+      tagPrefix = "v.";
```

Comment:
I thought this was a typo...but nope! That's so weird lol

Reply:
https://github.com/NixOS/nixpkgs/blob/a22cbab59462ff5e754db0c7691aa5ed4eb7183e/pkgs/common-updater/unstable-updater.nix#L18-L20

- `tagFormat` because upstream has tags that we must ignore: `Pre-release-shadPS4-<YYYY-MM-DD>-<commit-rev>`
- `tagPrefix` to strip the static `v.` part from the tag
- If getting a reasonable version identifier from the tag was more tricky, `tagConverter` instead of `tagPrefix`
https://github.com/NixOS/nixpkgs/blob/a22cbab59462ff5e754db0c7691aa5ed4eb7183e/pkgs/by-name/ch/chirp/package.nix#L69-L71